src/main/java/uk/gov/hmcts/reform/civil/model/FlightDelay.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.civil.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.civil.model.common.DynamicList;
+import uk.gov.hmcts.reform.civil.model.defaultjudgment.CaseLocationCivil;
+
+import java.time.LocalDate;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlightDelay {
+
+    private DynamicList flightDetailsAirlineList;
```

Comment:
If we channge name of pojo to "FlightDelayDetails", we can remove "flightDetails" part from variables names. For example one varialbe will be read "flightDelayDetails.airlineList" than "flightDelay.flightDetailsAirlineList" after changes.